functions/concepts/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Google Inc.
```

Comment:
I think we have a precedent that Dockerfiles do not require a copyright notice. I'll approve this but feel free to remove the notice before merging if you want.